athenian/types/types.go

```diff
@@ -23,6 +23,9 @@ type Database interface {
 	GetAccount(ctx context.Context, id AccountID) (*Account, error)
 	GetAccountBySecret(ctx context.Context, secret string) (*Account, error)
 	ListAccounts(ctx context.Context) ([]Account, error)
+	// DEV-3198
+	SetAccountBeta(ctx context.Context, id AccountID) error
```

Comment:
I think this is too specific. I propose to implement `SetAccountFeature` instead and add a constant for Beta feature flag.

Reply:
Done.